ports/aws-sdk-cpp/portfile.cmake

```
@@ -19,6 +19,22 @@ include(${CMAKE_CURRENT_LIST_DIR}/compute_build_only.cmake)
 
 if(VCPKG_TARGET_IS_OSX OR VCPKG_TARGET_IS_IOS)
     set(rpath "@loader_path")
+elseif (VCPKG_TARGET_IS_ANDROID)
+    vcpkg_find_acquire_program(GIT)
+
+    set(EXTRA_ARGS "-DTARGET_ARCH=ANDROID"
+            "-DGIT_EXECUTABLE=${GIT}"

```

```suggestion
    set(EXTRA_ARGS "-DTARGET_ARCH=ANDROID"
            "-DGIT_EXECUTABLE=--invalid-git-executable--"
```

Because we are providing prebuilt dependencies, the build should not actually need to use `git` (if it does, it's a bug!)